data-esnext.js

```diff
@@ -1,4 +1,4 @@
-var browsers = require('./esnext-browsers');
+exports.browsers = require('./esnext-browsers');
```

Comment:
@ljharb `build.js` expect `browsers` property to exist. In this PR I merged only es6 and esnext browsers. Browsers for es5, Intl and non-standard are still separate arrays. I agree that we should just import `browsers.js` into `build.js` but I think it could be done on some of next steps